tmva/sofie/inc/TMVA/ROperator_LeakyRelu.hxx

```diff
@@ -0,0 +1,71 @@
+#ifndef TMVA_SOFIE_ROPERATOR_LeakyRelu
+#define TMVA_SOFIE_ROPERATOR_LeakyRelu
+
+#include "TMVA/SOFIE_common.hxx"
+#include "TMVA/ROperator.hxx"
+#include "TMVA/RModel.hxx"
+
+#include <sstream>
+
+namespace TMVA{
+namespace Experimental{
+namespace SOFIE{
+
+template <typename T>
+class ROperator_LeakyRelu final : public ROperator
+{
+
+private:
+
+   std::string fNX;
+   std::string fNY;
+   std::vector<size_t> fShape;
+
+public:
+   ROperator_LeakyRelu(){}
+   ROperator_LeakyRelu(std::string nameX, std::string nameY):
+      fNX(UTILITY::Clean_name(nameX)), fNY(UTILITY::Clean_name(nameY)){}
+
+   std::vector<ETensorType> TypeInference(std::vector<ETensorType> input){
+      return input;
+   }
+
+   std::vector<std::vector<size_t>> ShapeInference(std::vector<std::vector<size_t>> input){
+      auto ret = input; //suggest copy to compiler
+      return ret;
+   }
+
+   void Initialize(RModel& model){
+      if (model.CheckIfTensorAlreadyExist(fNX) == false){   //input must be a graph input, or already initialized intermediate tensor
+         throw std::runtime_error("TMVA SOFIE Leaky Relu Op Input Tensor is not found in model");
+      }
+      fShape = model.GetTensorShape(fNX);
+      model.AddIntermediateTensor(fNY, model.GetTensorType(fNX), fShape);
+   }
+
+
+   std::string Generate(std::string OpName){
+      OpName = "op_" + OpName;
+      if (fShape.empty()) {
+         throw std::runtime_error("TMVA SOFIE Transpose Leaky Relu called to Generate without being initialized first");
+      }
+      std::stringstream out;
+      int length = 1;
+      for(auto& i: fShape){
+         length *= i;
+      }
+      out << "\n//------ LEAKY RELU\n";
+      out << SP << "for (int id = 0; id < " << length << " ; id++){\n";
+      out << SP << SP << "tensor_" << fNY << "[id] = ((tensor_" << fNX << "[id] > 0 )? tensor_" << fNX << "[id] : (float)0.01 * tensor_"<< fNX<<"[id]);\n";
```

Comment:
Can we parameterise the alpha value instead of fixing it to 0.01 here?
I believe ONNX Operators standards expects an alpha attribute.
https://github.com/onnx/onnx/blob/6a43e97e025422d50f9d38be71d49ddee667742b/docs/Operators.md#LeakyRelu

Reply:
Hello, @sanjibansg and @lmoneta I have added the alpha attribute to this PR, Kindly review it and suggest any changes if needed. Thanks!